### PR TITLE
Kill php 5.5

### PR DESCRIPTION
**Pull request by GrahamCampbell**
:trollface:

### CODE REVIEW

**Comment by Ocramius** [comment]
Few things to be added:

 * `::class` where applicable
 * remove 5.5 from `.travis.yml`

**Comment by GrahamCampbell** [comment]
> ::class where applicable

That could have been done on 1.6.x though, since `::class` is new in php 5.5 rather than 5.6?

**Comment by GrahamCampbell** [comment]
> remove 5.5 from .travis.yml

Oops, of course.

**Comment by Ocramius** [comment]
> That could have been done on 1.6.x though, since `::class` is new in php 5.5 rather than 5.6?

Good point :-)

File: .travis.yml [hunk 1]
```diff
@@ -21,7 +20,7 @@ services:
 before_install:
     - if [[ $TRAVIS_PHP_VERSION != 'hhvm' ]] ; then pecl channel-update pecl.php.net; fi;
     - if [[ $TRAVIS_PHP_VERSION != 'hhvm' && $TRAVIS_PHP_VERSION != '7.0' ]]; then pecl install riak-beta; fi;
-    - if [[ $TRAVIS_PHP_VERSION =~ 5.[56] ]] ; then echo yes | pecl install apcu-4.0.10; fi;
+    - if [[ $TRAVIS_PHP_VERSION = '5.6' ]] ; then echo yes | pecl install apcu-4.0.10; fi;
```
Comment (stof) [inline .travis.yml]: APCU is available for PHP 7 too

**Comment by Ocramius** [comment]
Good to go, sorry for the delays.

Will also add PHP 7.1 to the build matrix.

**Pull request merged**